clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/VeniceChangeCoordinate.java

```diff
@@ -73,7 +73,7 @@ protected String getTopic() {
     return topic;
   }
 
-  protected PubSubPosition getPosition() {
+  public PubSubPosition getPosition() {
```

Comment:
Do we really need to make this public?  The coordinate I think is exposed to customers, I'd rather they knew as little as possible about the internals of this object.

Reply:
I think? I want to print out some offset during debugging, let me check